assets/css/homepage.scss

```diff
@@ -0,0 +1,220 @@
+
+// Variables ------------ //
+
+$text-width:      660px;
+$section-padding: 7.2rem;
+$font-lead:   2rem;
+
+// Styles --------------- //
+
+p {
+  font-family: $font-sans;
+  font-size: 1.7rem;
+}
+
+.skipnav {
+  padding: 10px 15px;
+  position: absolute;
+  top: -42px;
+  left: 0px;
+  color: $color-base;
+  background: transparent;
+  @include transition(top 1s ease-out, background 1s linear);
+  z-index: 100;
+  &:focus {
+    position: absolute;
+    left: 0px;
+    top: 0px;
+    background: white;
+    outline: 0;
+    @include transition(top .1s ease-in, background .5s linear);
+  }
+}
+
+.usa-intro {
+  max-width: $text-width;
+}
+
+.usa-section-examples {
+  @include media($medium-screen) {
+    .usa-width-five-sixths {
+      @include shift(1);
+    }
+  }
+}
+
+.usa-section {
+  padding: {
+    bottom: $section-padding;
+    top: $section-padding;
+  }
+
+  h2 {
+    font-size: 3rem;
+  }
+}
+
+.usa-button-block {
+  display: inline-block;
+  vertical-align: top;
+}
+
+.usa-cta {
+  clear: both;
+  display: block;
+}
+
+.usa-banner {
+  @include media ($medium-screen) {
+    background-image: url("../img/home/homepage_illustrations_flag_cropped.png");
+    background-position: right bottom;
+    background-repeat: no-repeat;
+  }
+  background-color: $color-primary-darkest;
+  background-image: none;
+  color: $color-white;
+  padding: {
+    bottom: $section-padding;
+    top: 2rem;
+  }
+
+  .usa-lead {
+    color: $color-white;
+    margin-bottom: 1rem;
+  }
+
+  h1 {
+    color: $color-white;
+
+    @media (min-width: $medium-screen) {
+      font-size: 5.6rem;
+      padding-top: 4.5rem;
+    }
+  }
+
+  .usa-text-small {
+    font-size: 1.3rem;
+    margin-top: 0.1em;
+    text-align: center;
+  }
+
+  .usa-button-outline-inverse {
+    margin-right: 0;
+  }
+}
+
+.usa-banner-link-top {
+  color: $color-white;
+  float: right;
+  font-weight: $font-bold;
+
+  &:visited {
+    color: $color-white;
+  }
+}
+
+.usa-banner-content {
+  max-width: 48rem;
+}
+
+.usa-section-alt {
+  background-color: $color-gray-lightest;
+}
+
+.usa-section-dark {
+  background-color: $color-gray-dark;
+  color: $color-white;
+
+  h2,
+  h3,
+  .usa-lead {
+    color: $color-white;
+  }
+
+  .usa-width-one-half {
+    h3 {
+      clear: none;
+      margin-top: 0;
+    }
+
+    h3, p {
+      overflow: hidden;
+    }
+  }
+
+  .usa-width-one-half-top {
+    @media (min-width: $medium-screen) {
+      margin-bottom: 7rem;
+    }
+  }
+}
+
+h1, h2, h3, h4, h5, h6 {
+  color: $color-primary-darker;
+  font-family: $font-serif;
+}
+
+h1, h2 {
+  margin-top: 0;
+}
+
+.usa-lead {
+  color: $color-gray-dark;
+  font-weight: 300;
+  font-size: $font-lead;
+  margin-bottom: 5rem;
+}
+
+.intro-text {
+  font-family: $font-serif;
+  margin-bottom: 4rem;
+}
+
+a,
+a:hover {
+  border-bottom: 0;
+}
+
+.usa-img-circle {
+  float: left;
+  margin-right: 3rem;
+}
+
+.usa-standlast {
+  p {
+    font-size: $h3-font-size;
+  }
+}
+
+.usa-link-external {
```

Comment:
@msecret Check out the design below... We'll be adding in this external link icon, so that's why I think the name should be external.

<img width="232" alt="screen shot 2015-09-10 at 1 28 21 pm" src="https://cloud.githubusercontent.com/assets/5249443/9800109/fbc2cf32-57bf-11e5-8bd5-383d0ece35b0.png">